lib/iris/cube.py

```diff
@@ -2124,7 +2124,14 @@ def __repr__(self):
     def _repr_html_(self):
         from iris.experimental.representation import CubeRepresentation
         representer = CubeRepresentation(self)
-        return representer.repr_html()
+        # Catch exceptions in the html repr rather than passing them,
+        # and so fall back to the default cube repr.
+        try:
+            result = representer.repr_html()
+        except (IndexError, ValueError):
```

Comment:
My original preference was to catch any exceptions, but I see that's [bad practice](https://github.com/SciTools/iris/pull/3369). Is this too broad a range of exceptions, too narrow, or just right?